code/modules/vending/monkey.dm

```diff
@@ -0,0 +1,34 @@
+
+/obj/machinery/vending/monkey
+	name = "\improper Monkey Vendor"
+	desc = "A monkey vendor. Strange noises can be heard from inside."
+	icon_state = "valuechimp"
+	products = list(/obj/item/vended_monkey = 6)
+	refill_canister = /obj/item/vending_refill/monkey
+	payment_department = ACCOUNT_MED
+	cost_multiplier_per_dept = list(ACCOUNT_SCI = 0)
+
+// temporary monkey item that is dispensed onto the floor and then deletes itself, spawning a monkey
+/obj/item/vended_monkey
+	name = "vended monkey"
+	desc = "If you're seeing this, the coders made a mistake!"
+	item_flags = ABSTRACT // this stops it being placed into the user's hand by failing a can_put_in_hand check
+	icon = 'icons/obj/items_and_weapons.dmi'
+	icon_state = "valuechimp"
+
+/obj/item/vended_monkey/Initialize()
+	. = ..()
```

Comment:
Is it completely necessary to do a full init and then give it a hint to drop everything?